examples/sycl/pvc/pvc_bfloat_dpas_gemm_cute.cpp

```diff
@@ -46,13 +46,11 @@
 #include "cutlass/util/reference/device/tensor_compare.h"
 
 template <typename T>
-static void fill_matrix(std::vector<T> &M)
+static void fill_matrix(std::vector<T> &vector)
 {
-  std::random_device dev;
-  std::mt19937 rng(dev());
-  std::uniform_real_distribution<float> dist(1.0, 2.0);
-  std::generate(std::begin(M), std::end(M), [&]
-                { return static_cast<T>(dist(rng)); });
+  std::generate(std::begin(vector), std::end(vector), [&] {
+      return static_cast<T>( (rand() / double(RAND_MAX)) );
```

Comment:
do you need to cast to double?

Reply:
We can use `float` too. `double` is what is being used in all the other examples that use `rand() / RAND_MAX` for initialisation.